src/map/packets/message_system.h

```diff
@@ -26,6 +26,16 @@
 
 #include "basic.h"
 
+enum MSGSYSTEM_ID : uint16
+{
+    MSGSYSTEM_TRUST_NO_SEEKING_PARTY         = 296,   // You cannot use Trust magic while seeking a party.
+    MSGSYSTEM_TRUST_DELAY_NEW_PARTY_MEMBER   = 297,   // While inviting a party member, you must wait a while before using Trust magic.
+    MSGSYSTEM_TRUST_MAXIMUM_NUMBER           = 298,   // You have called forth your maximum number of alter egos.
+    MSGSYSTEM_MSGSYSTEM_TRUST_ALREADY_CALLED = 299,   // That alter ego has already been called forth.
+    MSGSYSTEM_TRUST_NO_ENMITY                = 300,   // You cannot use Trust magic while having gained enmity.
+    MSGSYSTEM_TRUST_SOLO_OR_LEADER           = 301,   // You cannot use Trust magic unless you are solo or the party leader.
+};
+
```

Comment:
It doesn't _need_ to be here, but this is following the pattern from basic messages since everything has to flow through core to be able to have the right packet sent out

Reply:
Ah ok, got ya, ok carry on lol

Reply:
and wow totaly missed that MSGSYSTEM_MSGSYSTEM lol, glad you caught that